### PR TITLE
yoo

### PR DESCRIPTION
**Pull request by Ece-Kahraman**
description

### CODE REVIEW

**Comment by aysekelleci** [comment]
deneme

**Comment by Ece-Kahraman** [comment]
> deneme
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2041386626 

never gonna let you down

**Comment by aysekelleci** [comment]
> deneme
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2041386626 

heyyo

**Comment by Ece-Kahraman** [comment]
**ACTIVE**: <p>hello</p>

**Comment by Ece-Kahraman** [comment]
> **ACTIVE**: <p>hello</p>
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2044763642 

sup

**Comment by aysekelleci** [comment]
> **ACTIVE**:
> hello

hi

**Comment by aysekelleci** [comment]
> deneme
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2041386626 

sdkfl

**Comment by aysekelleci** [comment]
**ACTIVE**: <p>sdfghn</p>

**Comment by aysekelleci** [comment]
> **ACTIVE**: <p>sdfghn</p>
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2066993971 

reply

File: README.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
 # HubReview
 
+This is the source code of our project.
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+hello
+BBBBBBBBBBBBB
```
Comment (vIremAydin) [inline README.md:6]: none(non-blocking): hell no

**Review by vIremAydin — CHANGES_REQUESTED** [review]

**Comment by vIremAydin** [comment]
> deneme
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2041386626 

deneme 11

**Comment by vIremAydin** [comment]
**ACTIVE**: <p><strong>the bold type</strong></p>

File: README.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+cececececec
+yeter artık daha editlemek istemiyorum
+bu son mu yoksa
+ulan rate limiti aştık
+alperin hesabıııı
+nolur ol artık ağlıycam
+check control
 TODO
 allam sana geliyorum
```
Comment (vIremAydin) [inline README.md:15]: question(blocking): lol

**Review by vIremAydin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vIremAydin) [reply README.md:6]: qszq

**Comment by aysekelleci** [comment]
> 
> **ACTIVE**: <p><strong>the bold type</strong></p>
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2072151987 

hello
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2072152205 

reply

**Comment by vIremAydin** [comment]
> 
> **ACTIVE**: <p><strong>the bold type</strong></p>
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2072151987 

hello
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2072152205 

replied

**Comment by vIremAydin** [comment]
> **ACTIVE**: <p><strong>the bold type</strong></p>
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2072151987 

you are great

**Comment by aysekelleci** [comment]
> 
> 
> **ACTIVE**: <p><strong>the bold type</strong></p>
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2072151987 

hello
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2072152205 

reply
> https://github.com/hubreviewapp/hubreview/pull/127#issuecomment-2072232468 

sljakdj

**Pull request closed without merging**